tree_migration_milestone1.js

```diff
@@ -1,19 +1,19 @@
 require("dotenv").config();
 const { knex } = require("./database/knex");
-
-var ProgressBar = require("progress");
+const Writable = require('stream').Writable;
+const ws = Writable({ objectMode: true });
+const ProgressBar = require("progress");
 
 async function migrate() {
   try {
     const base_query_string =
-      "from public.trees t left join field_data.raw_capture r on t.uuid = r.id::text where t.approved=true and r.id is null";
-
-    const count = await knex.select(knex.raw(`count(*) ${base_query_string}`));
-    const query_stream = knex.raw(`select t.* ${base_query_string}`).stream();
-
+      "select t.* from public.trees t left join field_data.raw_capture r on t.uuid = r.id::text where t.approved=true and r.id is null";
```

Comment:
This full query helps me to run partial migration runs by limiting the number of records selected. If you look at the rowCountResult, I am using this as a subquery. We have to use sub-query because `select`s with `count`, `where` and `limit` is not allowed in SQL.